Soomla/Assets/Plugins/Soomla/Profile/social-providers/FBSocialProvider.cs

```diff
@@ -161,18 +186,55 @@ public FBSocialProvider ()
 		/// <param name="success">Callback function that is called if the story update was successful.</param>
 		/// <param name="fail">Callback function that is called if the story update failed.</param>
 		/// <param name="cancel">Callback function that is called if the story update was cancelled.</param>
-		public override void UpdateStory(string message, string name, string caption,
+		public override void UpdateStory(string message, string name, string caption, string description,
 		                                 string link, string pictureUrl, SocialActionSuccess success, SocialActionFailed fail, SocialActionCancel cancel) {
-
-//			checkPermission("publish_actions", ()=> {
-				FB.Feed(
-					link: link,
-					linkName: name,
-					linkCaption: caption,
-					linkDescription: message,
-					picture: pictureUrl,
-					callback: (FBResult result) => {
+			checkPermission("publish_actions", ()=> {
```

Comment:
@EugeneButusov, do you really need it again? (Looks like degradation regarding https://github.com/vedi/unity3d-profile/commit/c649e928cdf33ebcf7e54828e2a59683554bd7e3#diff-f636b4f7159b314a47eb7964daef81ca).